### PR TITLE
Monitor child org doc update

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/manage/manage-subscription/create-manage-orgs-service-providers.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ You can view the aggreagte usage for all child orgs across usage category in the
 * **Infrequent Storage**. Credits used for log storage in the Infrequent tier.
 * **Promotional categories**. For more information, see [Monitoring Promotional Credit usage](cloud-flex-credits-accounts.md#monitoring-promotional-credit-usage).
 
-By clicking the **Download Report** button, you can download the org usage data in csv format for further analysis and reporting. You can also download the detailed child org usage data in csv format by clicking the **Download Detail Credit Usages** from the kebab icon next to the Download Report button.
+By clicking the **Download Report** button, you can download the org usage data in csv format for further analysis and reporting. You can also download the detailed child org usage data in csv format by clicking the **Download Detailed Credit Usages** from the kebab icon next to the Download Report button.
```
Comment (jpipkin1) [inline docs/manage/manage-subscription/create-manage-orgs-service-providers.md:161]:
```suggestion
By clicking the **Download Report** button, you can download the org usage data in csv format for further analysis and reporting. You can also download the detailed child org usage data in csv format by clicking **Download Detailed Credit Usages** from the kebab icon next to the Download Report button.
```

Comment (jpipkin1) [inline docs/manage/manage-subscription/create-manage-orgs-service-providers.md:161]:
```suggestion
By clicking the **Download Report** button, you can download the org usage data in csv format for further analysis and reporting. You can also download the detailed child org usage data in csv format by clicking **Download Detailed Credit Usages** from the kebab icon next to the Download Report button.
```

File: docs/manage/manage-subscription/create-manage-orgs.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ You can view the aggreagte usage for all child orgs across usage category in the
 * **Infrequent Storage**. Credits used for log storage in the Infrequent tier.
 * **Promotional categories**. For more information, see [Monitoring Promotional Credit usage](cloud-flex-credits-accounts.md#monitoring-promotional-credit-usage).
 
-By clicking the **Download Report** button, you can download the org usage data in csv format for further analysis and reporting. You can also download the detailed child org usage data in csv format by clicking the **Download Detail Credit Usages** from the kebab icon next to the Download Report button.
+By clicking the **Download Report** button, you can download the org usage data in csv format for further analysis and reporting. You can also download the detailed child org usage data in csv format by clicking the **Download Detailed Credit Usages** from the kebab icon next to the Download Report button.
```
Comment (jpipkin1) [inline docs/manage/manage-subscription/create-manage-orgs.md:129]:
```suggestion
By clicking the **Download Report** button, you can download the org usage data in csv format for further analysis and reporting. You can also download the detailed child org usage data in csv format by clicking **Download Detailed Credit Usages** from the kebab icon next to the Download Report button.
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812. I had only a couple of suggestions. 

Forgive me, but I also updated the numbered list layout so there wasn't a separate step for every field on the dialogs (see commit [Update numbered list](https://github.com/SumoLogic/sumologic-documentation/pull/3502/commits/6ca30e2d05f666eefb385085919e61f9517f0fd9)). I hope that's okay with you.

**Comment by JV0812** [comment]
@jpipkin1 Thanks for this John. It looks good now.

**Pull request merged**